imports/plugins/core/dashboard/client/containers/packageListContainer.js

```diff
@@ -15,9 +15,9 @@ function handleShowPackage(event, app) {
 }
 
 function composer(props, onData) {
-  const settings = Reaction.Apps({ provides: "settings", enabled: true }) || [];
-
-  const dashboard = Reaction.Apps({ provides: "dashboard", enabled: true })
+  const roles = Roles.getRolesForUser(Meteor.userId(), Reaction.getShopId());
```

Comment:
Maybe this should be move into the `Reacton.Apps` method. so any method that calls reaction apps without an `audience` will automatically be filtered.